### DISCUSSION TITLE
Drawing slows down every time the computer is turned on

### DISCUSSION THREAD

**Discussion by lrlbh** [Chia-Network/bladebit#102] [Q&A]
If you don't run the software again it's going to keep slowing down, no matter how many diagrams you create
If you rerun the software, it may get faster and continue to run at a faster rate  

It's 17-18 minutes on the fast side
The slow time is 19-20 minutes

(2697v2) * 2
(32G 1866) *14

**Comment by lrlbh** [comment]
Has anyone had this problem?

**Reply by bladeuserpi** [reply]
It seems sensitive about NUMA and Linux aggressive use of buffer cache.
-Avoid mv/cp while it is running (dd with directio is OK)
-Does performance recover with this?
  time echo 3 >/proc/sys/vm/drop_caches ; time sync

**Reply by lrlbh** [reply]
I'm using “dd”
echo 3 >/proc/sys/vm/drop_caches ;  # It's useless

My current solution is to run "-n 1" 3 times on startup and then run "-n 9999" once.
